src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -273,6 +273,17 @@
     </StringProperty.DataSource>
   </StringProperty>
 
+  <!-- TODO create fwlink for this HelpUrl -->
+  <BoolProperty Name="ProduceReferenceAssembly"
+                DisplayName="Reference assembly"
+                Description="Produce a reference assembly containing the public API of the project."
+                HelpUrl="https://docs.microsoft.com/en-us/dotnet/standard/assembly/reference-assemblies"
+                Category="Output">
+    <BoolProperty.DataSource>
+      <DataSource HasConfigurationCondition="False" />
```

Comment:
Looking at `HasConfigurationCondition`, it might make sense if we flip the default to false so controls need to opt into dimensionality. Also, by the name of this, it isn't obvious it has to do with configuration variants/dimensions. Maybe something like `IsVariedByConfiguration`?

Reply:
I'm open to flipping the default if most properties on this page are not configured. Feel free to submit a PR. You set the default in the `Rule`-level `DataSource`.

`HasConfigurationCondition` is an MSBuild API which has been public for forever and cannot be changed. If you think of the property it creates though, it does end up having a configuration condition (on the property group), so the name does make some sense.

To be clear, the `Rule` object model is used in many places throughout VS.